### PR TITLE
Allow installed modules translation from translation page

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.0.x
| Description?  | Allow installed modules translation from translation page
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1838
| How to test?  | Select modules in the list, choose the module you want to translate and check if you are redirected to the right page. When you're translating a module which use the new translation system you must be redirected to the new translation page, otherwise, it will be the old one with the right module selected.

### CODE REVIEW

File: src/PrestaShopBundle/Translation/Provider/ModulesProvider.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?php
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Translation/Provider/ModulesProvider.php]: would you mind to add a test for this provider ? all providers are already tested, you can look at others tests it's really easy to to and time saver for the future :smile_cat:

File: src/PrestaShopBundle/Tests/Translation/Provider/ModulesProviderTest.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            ->getMock()
+        ;
+
+        self::$resourcesDir = __DIR__.'/../../resources/translations';
+        $this->provider = new ModulesProvider($loader, self::$resourcesDir);
+    }
+
+    public function testGetMessageCatalogue()
+    {
+        // The xliff file contains 38 keys
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Tests/Translation/Provider/ModulesProviderTest.php]: This line should be removed

Comment (maximebiloe) [inline src/PrestaShopBundle/Tests/Translation/Provider/ModulesProviderTest.php]: 😅

**Review by mickaelandrieu — APPROVED** [review]

**Comment by aleeks** [comment]
Thank you @maximebiloe

**Pull request merged**